benchmark.go

```diff
@@ -170,9 +214,6 @@ func runBenchmark(out output, logger *zap.Logger, allOpts Options, resolved reso
 		}
 	}
 
-	// TODO: Support streaming updates.
-	// TOOD: Aggregate per-backend state for JSON output in future.
```

Comment:
I don't have context on these TODOs, but seems they should stay as is?

Reply:
Yes, I think so, let's leave these TODOs